components/list/list-item-drag-handle.js

```diff
@@ -0,0 +1,196 @@
+
+import '../button/button-icon.js';
+import '../icons/icon.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { buttonStyles } from '../button/button-styles.js';
+import { getFirstFocusableDescendant } from '../../helpers/focus.js';
+
+const keyCodes = Object.freeze({
```

Comment:
I think at least ESC is used for focus-trap. Maybe it'd be good to have a key-codes module?

`export const ESC = 27;
`import { ESC } from '../../helpers/key-codes';`

Reply:
I am thinking the same thing. Because I had to code this up in 2 places just for this PR. The aria-grid story will also need keys.

Reply:
I've considered this in the past.  The only reason I didn't was to avoid an extra import.  🤷‍♂️ I would not be opposed to it, and certainly there are a few places that define key codes for same purposes, as well as mocking key events in tests.

Reply:
I made the new file and updated the readme. I am instead of adding all the key codes at once I figured people can add them as the need them.